elasticsearch/src/main/java/org/hibernate/search/elasticsearch/settings/impl/translation/JsonPassThroughAnalysisDefinitionFactory.java

```diff
@@ -43,13 +43,7 @@ public String getType() {
 
 	@Override
 	public D create(Map<String,String> parameters) {
-		D result;
-		try {
-			result = targetClass.newInstance();
-		}
-		catch (InstantiationException | IllegalAccessException e) {
-			throw new AssertionFailure( "Unexpected failure while instanciating a definition", e );
-		}
+		D result = ClassLoaderHelper.untypedInstanceFromClass( targetClass, "Analyzer definition" );
```

Comment:
"Analyzer definition" => "analysis definition".
This encompasses tokenizer definitions, token filter definitions, char filter definitions as well as analyzer definitions.